### ISSUE TITLE
Issue with localization and custom validator

### ISSUE THREAD

**Issue by vincentjames501** [formvalidation/formvalidation#1343]
With the latest version (0.6.0), I'm getting the following error:

```
 TypeError: Cannot read property 'default' of undefined
    at FormValidation.Base._getMessage (/components/form.validation/dist/js/formValidation.js:553:79)
```

It looks like my custom validator is the problem.

```javascript
// Removed code from here, but this code yields the same issue
        FormValidation.Validator.remoteErrorValidator = {
            validate: function (validator, $field, options) {
                return {
                    valid: true,
                    message: ''
                }
            }
        };
```

I think the problem is with this switch statement:

```javascript
            switch (true) {
                case !!this.options.fields[field].validators[validatorName].message:
                    return this.options.fields[field].validators[validatorName].message;
                case !!this.options.fields[field].message:
                    return this.options.fields[field].message;
                case !!FormValidation.I18n[this.options.locale][validatorName]['default']:
                    return FormValidation.I18n[this.options.locale][validatorName]['default'];
                default:
                    return this.options.message;
            }
```

This statement ```FormValidation.I18n[this.options.locale][validatorName]['default']``` should be more robust and make sure that an object exists before accessing `default` on it otherwise the ```this.options.message``` default statement will never be reached.

**Comment by nghuuphuoc** [comment]
Your own validator seems good, because all the examples on [Developing custom validator](http://formvalidation.io/developing/) page work fine.
Please create a [jsfiddle](http://jsfiddle.net/) for it.

---
_If FormValidation is useful to you, please Star this project, and [Buy it](http://formvalidation.io/download/) :heart:_

**Comment by vincentjames501** [comment]
I think it's because in your examples you're providing a default message when bootstrapping the form element. Do you have a good jsfiddle starting point that already includes 0.6.0. Don't have much time to reproduce.

**Comment by nghuuphuoc** [comment]
It might be caused by ```FormValidation.I18n[this.options.locale][validatorName]['default']```.
Do you set the ```locale``` option when calling ```.formValidation()```?

**Comment by nghuuphuoc** [comment]
@vincentjames501 
The issue is fixed in the latest code. The fix will be available in the next v0.6.1!
